src/lib/connect/ali.ts

```diff
@@ -75,11 +75,13 @@ function bindEventHandler() {
 		} else {
 			const reader = new FileReader()
 			reader.addEventListener('load', () => {
-				let data = reader.result
+				if (reader.result instanceof ArrayBuffer) {
+					proxy.push(Buffer.from(reader.result))
 
-				if (data instanceof ArrayBuffer) data = Buffer.from(data)
-				else data = Buffer.from(data, 'utf8')
-				proxy.push(data)
+					return
+				}
+
+				proxy.push(Buffer.from(reader.result, 'utf-8'))
```

Comment:
Since there was a typing issue here, I made a minor refactoring to make the code lighter.